base_edifact/tests/test_base_edifact.py

```diff
@@ -23,6 +23,11 @@ def test_pydifact_obj(self):
         obj = self.base_edifact_model.pydifact_obj(edifact_docu)
         # [1]: to get the list messages, [0]: to get the first list value of the segments
         self.assertEqual(obj[1]["segments"][0]["BGM"][1], "1AA1TEST")
+        # Test with format latin-1
```

Comment:
better having atomic tests: add `test_pydifact_obj_latin1()` for testing latin-1